variables.tf

```diff
@@ -70,3 +70,8 @@ variable "labels" {
   description = "The resource labels to represent user provided metadata."
   default     = {}
 }
+
+variable "enable_apis" {
```

Comment:
This change should have added an entry in `README.md`, does `make generate_docs` create any changes?